apps/web/app/(navigation)/budgets/[id]/TransactionTableController.tsx

```diff
@@ -43,13 +43,14 @@ const TransactionTableController = ({ budget }: { budget: BudgetFixed }) => {
   const setSorting = (column: string) => console.log(column);
   const { data: session } = useSession();
   const categoryFilterState = useAtomValue(categoryFilterAtom);
+  const pageSize = parseFloat(getPageSizeValue);
```

Comment:
this should be changed to `parseInt`. it is kept in local storage, so it can be changed by user. right now, I can change it to 4.5 and it is sent like this to backend. Transaction table crashes then.

```suggestion
  const pageSize = parseInt(getPageSizeValue);
```


![image](https://github.com/intive/patronage2023-js/assets/80450315/40e5cba7-0a6b-48fd-bca2-27d1c49c7075)